.storybook/preview.ts

```diff
@@ -20,6 +20,10 @@ const preview: Preview = {
         dark: 'dark'
       },
       defaultTheme: 'light'
+    }),
+    (story) => ({
+      components: { story },
+      template: '<div class="dark:bg-gray-900 px-6 py-4"><story /></div>'
```

Comment:
```suggestion
      template: '<div class="bg-gray-50 p-10 dark:bg-gray-900"><story /></div>'
```
I would add a bit more padding, e.g. to test tooltips position

Reply:
No problem there